package.json

```diff
@@ -34,14 +34,16 @@
     "lodash.uniqby": "^4.7.0",
     "onecolor": "^3.0.5",
     "prop-types": "^15.6.2",
-    "react": "^16.8.3",
+    "react": "^16.8.6",
     "react-blockies": "^1.3.0",
     "react-container-dimensions": "^1.3.3",
-    "react-dom": "^16.8.4",
+    "react-dom": "^16.8.6",
```

Comment:
Just to confirm: is this only to align the two versions, or also because the previous versions were causing issues?

Reply:
Only to align I guess as it was a freebie when `npm i`nstalling